File: 1952-three-divisors/1952-three-divisors.cpp
class Solution {
public:
    bool isThree(int n) {
        int count=0;
        for (int i=1;i<n+1;i++){
            if( n%i==0){
                if (count<3)
                    count+=1;
                else
                    return false;
            }
            
        }
        if(count==3)
                return true;
            else 
                return false;
        
    }
};

File: 1952-three-divisors/1952-three-divisors.py
class Solution:
    def isThree(self, n: int) -> bool:
        count=0
        for i in range(1,n+1):
            if n%i==0:
                if count<3:
                    count+=1
                else:
                    return False
        return True if count==3 else False